src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer.Common/LanguageServerConstants.cs

```diff
@@ -3,77 +3,77 @@
 
 namespace Microsoft.AspNetCore.Razor.LanguageServer.Common;
 
-    public static class LanguageServerConstants
-    {
-        public const int VSCompletionItemKindOffset = 118115;
+public static class LanguageServerConstants
+{
+    internal const int VSCompletionItemKindOffset = 118115;
```

Comment:
This file is a great example of where I'd love to have a new DLL serve as an external access layer, so its much easier to see exactly what surface area is exposed, and also harder (if not impossible) to accidentally widen that area.

Of course that could come as a follow up, not in this PR, since this one does the hard work of at least identifying what that surface area is.